source/frontend/src/features/leases/add/LeaseDetailSubForm.tsx

```diff
@@ -154,11 +156,42 @@ export const LeaseDetailSubForm: React.FunctionComponent<ILeaseDetailsSubFormPro
           </SectionField>
         </Col>
         <Col>
-          <SectionField label="Expiry date">
+          <SectionField
+            label="Expiry"
+            required={statusTypeCode === ApiGen_CodeTypes_LeaseStatusTypes.ACTIVE}
```

Comment:
I think this field is never required - this rule only applies to the commencement date on the agreement.

Reply:
This was part of the conversation with Julian. Active leases must have a commencement and expiry dates